.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           paths:
               - src/NiMARE
       - restore_cache:  # ensure this step occurs *before* installing dependencies
-          key: deps9-{{ checksum "nimare/info.py" }}-{{ checksum "setup.py" }}
+          key: deps10-{{ checksum "nimare/info.py" }}-{{ checksum "setup.py" }}
```

Comment:
You'll want to do that for all of the other instances of `deps9` in this file.

BTW I never actually thought to use that before! I normally make a dummy change to `setup.py`, which I now realize is a very silly approach.

Reply:
actually, I do not need to change that, I changed `nimare/info.py`, so that should actually update the cache, forgot to revert this change but glad you learned something from it.